src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
```

Comment:
It's not recommended to use the '*' selector, as it can impact performance. It's better to set styles only for elements that require them. (CSS checklist)